bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Wavelength.scala

```diff
@@ -103,6 +70,10 @@ object Wavelength {
   implicit val WavelengthOrdering: scala.Ordering[Wavelength] =
     scala.Ordering.by(_.toNanometers)
 
-
+  /** @group Implicit Conversions */
+  implicit def fromLength(l: Length): Wavelength = {
```

Comment:
Can we can add our own `Int` pimping to get `.nm` and `.microns` that convert to `Wavelength` and still have the concise code?